### PR TITLE
added Michael dinner recipe, but i have an issue about the image it d…

### PR DESCRIPTION
**Pull request by hebaDawahra**
…idn't appear

### CODE REVIEW

File: recipe-chicken-with-apricot.html [hunk 1]
```diff
@@ -14,7 +14,7 @@
         <div class="image-section">
             <h3>Dinner: Chicken With Apricot</h3>
             <a href="./images/recipe-chicken-with-apricot.jpg">Click For Larger Image</a>
-            <img src="./images/recipe-chicken-with-apricot.jpg" alt="Recipe: Chicken with Apricots">
+            <img src="./images/recipe-chicken-with-apricot.jpg">
```
Comment (Fred-Khan) [inline recipe-chicken-with-apricot.html]: Please re-add the alt attribute to the image.

**Review by Fred-Khan — CHANGES_REQUESTED** [review]
Please make the changes to **recipe-chicken-with-apricot.html** as requested.

File: recipe-chicken-with-apricot.html (hunk 1 above)
Comment (hebaDawahra) [reply recipe-chicken-with-apricot.html]: done, I've re-added it

File: images/recipe-chicken-with-apricot 8.jpg
Comment (Fred-Khan) [inline images/recipe-chicken-with-apricot 8.jpg:1]: You are referencing **recipe-chicken-with-apricot.jpg** on line 5 in **dinnerList.js**. However, this file's name does not match the name you are referencing but there is already a file in images called **recipe-chicken-with-apricot.jpg**. Is this file required or should it be removed?

**Review by Fred-Khan — CHANGES_REQUESTED** [review]
- [x] Is the **recipe-chicken-with-apricot 8.jpg** file needed, or can it be removed from the repo?

File: images/recipe-chicken-with-apricot 8.jpg
Comment (hebaDawahra) [reply images/recipe-chicken-with-apricot 8.jpg:1]: i should remove  recipe-chicken-with-apricot 8.jpg

**Review by Fred-Khan — APPROVED** [review]
Thanks Heba.
